clang/utils/perf-training/CMakeLists.txt

```diff
@@ -15,7 +19,7 @@ if(LLVM_BUILD_INSTRUMENTED)
     )
```

Comment:
@petrhosek CLANG_PERF_TRAINING_DEPS was an existing variable that's used here.  I think it was meant to be added to cache files, but I added it to CMakeLists.txt file here so that it could be passed on the command line without modifying the cache files.

Reply:
Looks like I added that in https://reviews.llvm.org/D138974 but I'm not aware of it being used anywhere at the moment. I'd still prefer renaming it to `CLANG_PGO_TRAINING_DEPS` for consistency, especially that we're now adding it to cache.